pandas/io/sql.py

```diff
@@ -17,6 +17,7 @@
 )
 from functools import partial
 import re
+import sqlite3
```

Comment:
this adds about 1% to import time

Reply:
thanks @jbrockmendel ! yeah some of these will need adding to the blocklist, I'll take a closer look later

Reply:
Reverted code back.